### ISSUE TITLE
`isfixed` not defined

### ISSUE THREAD

**Issue by Octogonapus** [JuliaTesting/TestEnv.jl#26]
I am using Julia v1.6.1 and TestEnv.jl v1.4.1. When I activate my project and then run this code, I get the following error:
```julia
julia> TestEnv.activate()
ERROR: UndefVarError: isfixed not defined
Stacktrace:
 [1] activate(pkg::String)
   @ TestEnv ~/.julia/packages/TestEnv/kAms9/src/activate_set.jl:37
 [2] activate()
   @ TestEnv ~/.julia/packages/TestEnv/kAms9/src/activate_set.jl:7
 [3] top-level scope
   @ REPL[9]:1
```

**Comment by oxinabox** [comment]
Can you share the Project.toml that triggers this?

**Comment by Octogonapus** [comment]
This was in a private project but I managed to reproduce it with this public one: https://github.com/Octogonapus/CountDownLatches.jl
Manifest: https://pastebin.com/36VeKZtF

**Comment by willtebbutt** [comment]
Can confirm that the proposed fix works for me.

**Comment by oxinabox** [comment]
@willtebbutt, can you make a PR (the 1.4 branch) adding the fix and the tests?
(and then if all good I will port it to the others)

**Comment by Octogonapus** [comment]
Closed in v1.4.2
